kdwsdl2cpp/src/converter.h

```diff
@@ -41,6 +41,16 @@ namespace KWSDL
 class Converter
 {
 public:
+
+  struct DefaultAttributeValue
+  {
+    DefaultAttributeValue(): mIsBuiltin(false) {
```

Comment:
I'd suggest to add the two arguments to the constructor, to simplify the calling code.

Reply:
fixed